packages/c/canvasjs.json

```diff
@@ -1,16 +1,45 @@
 {
-  "name": "canvasjs",
-  "filename": "canvasjs.min.js",
-  "description": "CanvasJS is an easy to use HTML5 & JavaScript Charting library built on Canvas element.",
-  "homepage": "http://canvasjs.com/",
-  "keywords": [
-    "canvas",
-    "charts"
-  ],
-  "license": "Creative Commons Attribution-NonCommercial 3.0",
-  "authors": [
+  "name":"canvasjs",
+  "filename":"canvasjs.min.js",
+  "description":"CanvasJS is a JavaScript Charting Library for creating interactive charts & graphs for your web pages. Library supports a wide range of chart types including bar, line, area, pie, doughnut charts, etc. Charts work with all the popular Technologies & Frameworks like React, Angular, jQuery, PHP, etc.",
+  "homepage":"https://canvasjs.com/",
+  "license":"SEE LICENSE IN <LICENSE>",
+  "autoupdate":{
+    "source":"npm",
+    "target":"@canvasjs/charts",
+    "fileMap":[
+      {
+        "basePath":"",
+        "files":[
+          "*.js"
+        ]
+      }
+    ]
+  },
+  "authors":[
     {
-      "name": "fenopix"
+      "name":"fenopix"
     }
+  ],
+  "repository":{
+    "private":true
+  },
```

Comment:
As the CI error notes, cdnjs requires that the `repository` field be populated with an actual open-source repository.